src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=903180926&format=interactive",
+  sheets_gid="1829901599",
+  sql_file="as_attribute_by_year.sql"
+) }}
+
+Rispetto alla tendenza in [2019](../2019/resource-hints#the-as-attribute), abbiamo visto una rapida crescita nell'utilizzo di font e style con l'attributo `as`. Ciò è probabilmente correlato agli sviluppatori che aumentano la priorità dei CSS critici e combinano anche i font `preload` con `display:optional` per [migliorare](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit) Cumulative Layout Shift ([CLS](https://web.dev/cls/)).
+
+Tieni presente che omettere l'attributo `as` o avere un valore non valido renderà più difficile per il browser determinare la priorità corretta e in alcuni casi, come gli script, può anche causare il fetch della risorsa due volte.
+
+### L'attibuto `crossorigin`
+
+Con le risorse `preload` e `preconnect` che hanno CORS abilitato, come i fonts, è importante includere l'attributo `crossorigin`, in modo che la risorsa venga utilizzata correttamente. Se l'attributo `crossorigin` è assente, la richiesta seguirà la politica della single-origin rendendo così inutile l'uso del preload.
+
+{{ figure_markup(
+  caption="La percentuale di elementi con `preload` che usano `crossorigin`.",
+  content="16.96%",
+  classes="big-number",
+  sheets_gid="1185042785",
+  sql_file="attribute_usage.sql"
+) }}
+
+Le ultime tendenze mostrano che il 16,96% degli elementi che `preload` imposta anche `crossorigin` e si caricano in modalità anonima (o equivalente), e solo lo 0,02% utilizza il caso `use-credentials`. Questa velocità è aumentata in concomitanza con l'aumento del precaricamento dei font, come accennato in precedenza.
+
+```html
+<link rel="preload" href="ComicSans.woff2" as="font" type="font/woff2" crossorigin>
+```
+
+Tieni presente che i fonts precaricati senza l'attributo `crossorigin` verranno recuperati [due volte](https://web.dev/preload-critical-assets/#how-to-implement-relpreload)!
+
+### L'attributo `media`
+
+Quando è il momento di scegliere una risorsa da usare con schermi di dimensioni diverse, cerca l'attributo `media` con `preload` per ottimizzare le tue query multimediali.
+
+```html
+<link rel="preload" href="a.css" as="style" media="only screen and (min-width: 768px)">
+<link rel="preload" href="b.css" as="style" media="screen and (max-width: 430px)">
+```
+
+Vedere oltre 2.100 diverse combinazioni di media query nel dataset del 2020 ci incoraggia a considerare quanto sia ampia la varianza tra il concetto e l'implementazione del responsive design da un sito all'altro. I punti di interruzione sempre popolari `767px/768px` (resi popolari da Bootstrap tra gli altri) possono essere visti nei dati.
+
+### Best practices
```

Comment:
And this

Reply:
@chefleo did you see this? Hidden under a "Load more..." so you might have missed it.

Reply:
I search a bit around, and I am pretty confident `Best practices` now is a common known word in Italy 😄, so I think is good we don't translate this one. cc @chefleo @giopunt

Reply:
That's fine then to leave as is.

FYI, if we want to wrap it in lang tag (optional extra!) then can do this to preserver the anchor name without this attribute:

```suggestion
### <i lang="en">Best practices</lang> {best-practices}
```

But if it really is commonly known in Italy, then don't need to do this and can leave as is.